README.md

```diff
@@ -41,22 +29,32 @@ One of the very common use cases of WebXR is 180 and/or 360 photos or videos, bo
 
 
-### Implementation overview
+## Goals of the proposal
+The purpose of this document is to unlock the ability for User Agents to experiment with multiple layers. 
+* Introduce a concept of layers into WebXR spec and the way how layers should be provided from user's code to Browser;
+* Define ways to get layers capabilities, such as maximum number of supported layers, supported types of layers, etc;
+* Make sure the layers' part of the WebXR spec is extensible for any future additions and minimizing need for breaking changes in future. The modified API should be compatible with all User Agents whether they support multiple layers or not. 
+
+## Non-gials
+* This document doesn't provide definitions for any specific layer types.
+* Particular details of implementation will be provided in a different [document](details.md)
+
+
+## Proposed changes
+
+Current WebXR spec is written with layers in mind (see [XRLayer interface](https://immersive-web.github.io/webxr/#xrlayer-interface)), however, only a single layer ([baseLayer](https://immersive-web.github.io/webxr/#dom-xrrenderstate-baselayer)) is currently supported.
 
-A very high-level description of the implementation could be this:
+To support multiple layers the following changes are proposed:
 * Replace the [XRRenderState](https://immersive-web.github.io/webxr/#xrrenderstate-interface)'s [baseLayer](https://immersive-web.github.io/webxr/#dom-xrrenderstate-baselayer) with the sequence of [XRLayer](https://immersive-web.github.io/webxr/#xrlayer-interface)s;
 * Layers are drawn in the same order as they are specified in via `XRSession/updateRenderState`, with the
 0th layer drawn first. Layers are drawn with a "painter’s algorithm," with each successive layer potentially overwriting the destination layers whether or not the new layers are virtually closer to the
 viewer.
 * Introduce a way to determine which layers are supported and what is the maximum amount of the layers supported (fingerprinting?);
 * Add common properties to [XRLayer](https://immersive-web.github.io/webxr/#xrlayer-interface) (such as chromatic aberration or alpha blending flags);
 * Introduce a concept of "image source" for layers and a way to create / request image sources from the UA;
-* Optionally, change [XRWebGLLayer](https://immersive-web.github.io/webxr/#xrwebgllayer-interface) to comply with the new concepts of XRLayer and image source. Or, we can leave it as is and replace it with `XRLayerProjection` once we are ready;
-* Introduce different subtypes to [XRLayer](https://immersive-web.github.io/webxr/#xrlayer-interface) which will have all the necessary attributes for each layer.
+* Optionally, change [XRWebGLLayer](https://immersive-web.github.io/webxr/#xrwebgllayer-interface) to comply with the new concepts of XRLayer and image source. Or, we can leave it as is and replace it with a different layer once we are ready.
```

Comment:
For now, let's leave off these last two bullets entirely.  We can mention them verbally in the session tomorrow, but I think they're unfinished enough that it'll be distracting here.  Once they're in better shape, we can add the reference back if we think it's valuable to continue editing this document